src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="f">
+          <img
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <div class="nav">
+        <ul class="list">
+          <li>
+            <a
+              class="nav_link is-active"
+              href="e"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="e"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="e"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="ee"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="e"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="e"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="e"
```

Comment:
The href attribute in this anchor tag is set to 'e', which seems incorrect. Ensure it points to a valid URL or path.